<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JewelryClass</class>
 <widget class="QMainWindow" name="JewelryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Производство ювелирных изделий</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="technologist_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Технолог</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Администратор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customer_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Покупатель</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Курсовой проект выполнила Егорова К.О. 425 группа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите пользователя</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="jewelry.qrc"/>
 </resources>
 <connections/>
</ui>
